Realm/RLMObjectBase.mm

```diff
@@ -159,14 +156,42 @@ - (NSArray *)linkingObjectsOfClass:(NSString *)className forProperty:(NSString *
     }
     
     size_t col = prop.column;
-    NSUInteger count = _row.get_backlink_count(*table, col);
+    NSUInteger count = object->_row.get_backlink_count(*table, col);
     NSMutableArray *links = [NSMutableArray arrayWithCapacity:count];
     for (NSUInteger i = 0; i < count; i++) {
-        [links addObject:RLMCreateObjectAccessor(_realm, schema, _row.get_backlink(*table, col, i))];
+        [links addObject:RLMCreateObjectAccessor(object->_realm, schema, object->_row.get_backlink(*table, col, i))];
     }
     return [links copy];
 }
 
+
+FOUNDATION_EXTERN BOOL RLMObjectBaseAreEqual(RLMObjectBase *o1, RLMObjectBase *o2) {
```

Comment:
The definitions need to be declared `extern "C"` since they're in an obj-c++ file.